include/dba.php

```diff
@@ -691,7 +691,7 @@ static public function e($sql) {
 
 			self::close($stmt);
 
-		} while ((self::$dbo->errorno = 1213) && (--$timeout > 0));
+		} while ((self::$dbo->errorno == 1213) && (--$timeout > 0));
```

Comment:
Turn it around to avoid accidently assigning a variable:
`} while ((1213 == self::$dbo->errorno) && (--$timeout > 0));`